pointerevents/pointerevent_touch-action-pan-left-css_touch-manual.html

```diff
@@ -0,0 +1,114 @@
+<!doctype html>
+<html>
+    <head>
+        <title>touch-action: pan-left</title>
+        <meta name="assert" content="TA15.3 - With `touch-action: pan-left` on a swiped or click/dragged element, only panning on the x-axis left direction should be possible.">
+        <meta name="viewport" content="width=device-width">
+        <link rel="stylesheet" type="text/css" href="pointerevent_styles.css">
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="pointerevent_support.js"></script>
+        <style>
+            #target0 {
+            width: 700px;
+            height: 430px;
+            touch-action: pan-left;
+            }
+        </style>
+    </head>
+    <body onload="run()">
+        <h1>Pointer Events touch-action attribute support</h1>
+        <h4 id="desc">Test Description: Try to scroll element DOWN (drag up), then RIGHT (drag left), then LEFT (drag right). Tap Complete button under the rectangle when done. Expected: only pans in left direction.</h4>
```

Comment:
nit: "s/drag right/drag right as far as possible/" since you're asserting 0 (not just less than 200).  Here and the other tests that assert both scrollTop and scrollLeft equal 0.  Sorry if this change was in your previous patch and I missed it.